Helpers/StaticHelpers.gd

```diff
@@ -34,6 +34,14 @@ static func convert_from_range(value: float, from_min: float, from_max: float, t
 	return to_value
 
 
+static func get_mean_vector(positions: Array[Vector2]) -> Vector2:
+	if positions.size() <= 0:
+		return Vector2.ZERO
+	var mean_vector: Vector2 = Vector2.ZERO
+	for position in positions:
```

Comment:
missing typing for position. Isn't Godot complaining since we have typing as error ?

Reply:
I think that error doesn't work for for loops specifically. I'll fix